recipes/oceanspy/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.0.11" %}
+
+package:
+  name: oceanspy
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/o/oceanspy/oceanspy-{{ version }}.tar.gz
+  sha256: dea364ffbe152a3a1809486cea34a5d9e51fdc807af005d0cdadfdc443ad46b8
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
```

Comment:
can pytest be removed?

Reply:
Nope. It is a "setup" requirement :unamused: